index.js

```diff
@@ -158,4 +158,4 @@ RateLimit.prototype.storageSet = function (callback) {
   function onResult(err, res) {
     callback(err ? err : undefined);
   }
-}
\ No newline at end of file
+}
```

Comment:
Sorry for that, the GitHub editor did this to the last line ending.
Since the change was so small, I didn't bother to fork...